tests/unit/docker/hub-mwu-bad/bad-pythonfile/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM jinaai/jina:test-pip
 
-sadasdasdADD *.py mwu_encoder.yml ./
+ADD *.py mwu_encoder.yml ./
```

Comment:
this is bad on purpose to fail in the test

Reply:
This is the directory for the `bad-pythonfile` which already contains errors.
The `Dockerfile` can be correct in that case, can it?

Reply:
I reverted it in https://github.com/jina-ai/jina/pull/1958/commits/a47b970e35bec04f58f8b38de5f8c0c9324b6ef7